Source/Scripts/cheat_core_player.lua

```diff
@@ -108,147 +87,336 @@ function cheat:teleport_to(line)
-  places["TREASURE_7"] = "x:1860 y:1521 z:80"
-  places["TREASURE_8"] = "x:2332 y:1120 z:54"
-  places["TREASURE_9"] = "x:869 y:3279 z:19"
-  places["TREASURE_10"] = "x:1683 y:938 z:41"
-  places["TREASURE_11"] = "x:1445 y:1140 z:37"
-  places["TREASURE_12"] = "x:3175 y:335 z:136"
-  places["TREASURE_13"] = "x:3610 y:721 z:100"
-  places["TREASURE_14"] = "x:3692 y:1258 z:87"
-  places["TREASURE_15"] = "x:2942 y:1329 z:90"
-  places["TREASURE_16"] = "x:482 y:2578 z:20"
-  places["TREASURE_17"] = "x:769 y:2572 z:20"
-  places["TREASURE_18"] = "x:2494 y:2817 z:99"
-  places["TREASURE_19"] = "x:856 y:1335 z:18"
-  places["TREASURE_20"] = "x:740 y:3699 z:30"
-  places["TREASURE_21"] = "x:657 y:3141 z:41"
-  places["TREASURE_22"] = "x:600 y:608 z:158"
-  places["TREASURE_23"] = "x:1011 y:3972 z:51"
-  places["TREASURE_24"] = "x:903 y:3841 z:66"
-  places["TREASURE_25"] = "x:221 y:3474 z:77"
-  places["ANCIENT_TREASURE_1"] = "x:3872 y:886 z:157"
-  places["ANCIENT_TREASURE_2"] = "x:874 y:270 z:181"
-  places["ANCIENT_TREASURE_3"] = "x:3159 y:3840 z:167"
-  places["ANCIENT_TREASURE_4"] = "x:1723 y:778 z:74"
-  places["ANCIENT_TREASURE_5"] = "x:474 y:3869 z:40"
+  if places[cheat:toUpper(args)] ~= nil then
+    cheat:teleport(places[cheat:toUpper(args)])
+  else
+    local checkteste = "error"
+    for k,v in pairs(places) do
+      if string.find(k, cheat:toUpper(args)) then
+        checkteste = v
+      end
+    end
+    if checkteste ~= "error" then
+      cheat:teleport(checkteste)
+    else
+      cheat:logError("Invalid place - For a list of supported places type: 'cheat_teleport_to ?'")
+    end
+  end
+end
 
-  -- All bandit camp locations for RUIN Quest.
-  places["RUIN_1"] = "x:3385 y:633 z:60"
-  places["RUIN_2"] = "x:1724 y:708 z:69"
-  places["RUIN_3"] = "x:3212 y:1380 z:101"
-  places["RUIN_4"] = "x:1726 y:951 z:47"
-
-  -- -- All bandit camp locations for RAIDERS Quest.
-  places["RAIDERS_1"] = "x:2464 y:2195 z:150"
-  places["RAIDERS_2"] = "x:2668 y:3163 z:136"
-  places["RAIDERS_3"] = "x:1669 y:3257 z:53"
-  places["RAIDERS_4"] = "x:3615 y:3193 z:172"
-  places["RAIDERS_5"] = "x:2454 y:2362 z:100"
 
-  -- All bandit camp locations for INTERLOPERS Quest.
-  places["INTERLOPERS_1"] = "x:188 y:3266 z:106"
-  places["INTERLOPERS_2"] = "x:536 y:2840 z:87"
-  places["INTERLOPERS_3"] = "x:536 y:2395 z:32"
+-- ============================================================================
+-- cheat_tp_bc (BANDIT CAMPS)
+-- ============================================================================
+cheat.cheat_tp_bc_args = {
+  place = function(args,name,showHelp) return cheat:argsGetRequired(args, name, showHelp, "Teleport to bandit camp") end,
+}
 
+cheat:createCommand("cheat_tp_bc", "cheat:tp_bc(%line)", cheat.cheat_tp_bc_args,
+"Teleports the player to the given place. Supported places (case insensitive):\n$8RN_1, RN_2, RN_3, RN_4, RS_1, RS_2, RS_3, RS_4, RS_5, IL_1, IL_2, IL_3,\n$8BC_1, BC_2, BC_3, BC_4, BC_5, BC_6, BC_7, BC_8, BC_9, BC_10",
```

Comment:
Should change 1st sentence. "Teleports the player to the given bandit camp." so people know what this does.